products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp

```diff
@@ -44,7 +46,9 @@ namespace app::focus
         focusSessionsLeft     = allFocusSessionsCount;
 
         betweenSessionTimer = sys::TimerFactory::createSingleShotTimer(
-            app, "betweenSessionTimer", delayBetweenSessions, [this](sys::Timer &) { executeNextStep(); });
+            app, delayBetweenSessionsTimerName, delayBetweenSessions, [this]([[maybe_unused]] sys::Timer &) {
```

Comment:
```suggestion
            app, delayBetweenSessionsTimerName, delayBetweenSessions, [this]([[maybe_unused]] sys::Timer &t) {
```
Either `[[maybe_unused]]` or unnamed parameter :wink: